src/OrbitDB.js

```diff
@@ -30,17 +31,26 @@ let databaseTypes = {
 class OrbitDB {
   constructor(ipfs, directory, options = {}) {
     this._ipfs = ipfs
-    this.id = options.peerId || (this._ipfs._peerInfo ? this._ipfs._peerInfo.id._idB58String : 'default')
+    this.id = options.peerId
+    this.identity = options.identity
```

Comment:
Should identity be a required arg? I believe nothing on the log-level will work if there's no identity.

Reply:
Was trying to keep the interface the same, but if we are going to break things then yes I agree we should require identity.

Reply:
Agreed, we should try to not break things now and can leave this for later refactoring (knowing we'll break things then).